connect_deprish_to_ferc1.py

```diff
@@ -0,0 +1,214 @@
+        self.steam_cleaned_ferc1_raw = pd.read_pickle(
+            self.file_path_steam_ferc1, compression='gzip')
+        self.connects_ferc1_eia_raw = pd.read_pickle(
+            self.file_path_ferc1_eia, compression='gzip')
+        self.connects_deprish_eia_raw = pd.read_pickle(
+            self.file_path_deprish_eia, compression='gzip')
+
+    def _prep_plant_parts_eia(self):
+        self.plant_parts_eia = self.plant_parts_eia_raw.reset_index()
+
+    def _prep_connects_ferc1_eia(self):
+        # should this be done over in connect_ferc1_to_eia land?
+        # i think much of this and _prep_steam_cleaned_ferc1 should be moved.
+        self.connects_ferc1_eia = (
+            pd.merge(
+                self.connects_ferc1_eia_raw.reset_index()[
+                    ['record_id_ferc', 'record_id_eia']],
+                self.plant_parts_eia[
+                    ['record_id_eia', 'plant_part', 'report_date',
+                     'plant_id_pudl', 'utility_id_pudl']],
+                how='left')
+            .astype(connect_deprish_to_eia.prep_int_ids(
+                ['plant_id_pudl', 'utility_id_pudl', ]))
+        )
+
+    def _prep_connects_deprish_eia(self):
+        self.connects_deprish_eia = (
+            self.connects_deprish_eia_raw[
+                self.connects_deprish_eia_raw.record_id_eia.notnull()]
+            .pipe(pudl.helpers.convert_to_date)
+            .astype(connect_deprish_to_eia.prep_int_ids(
+                ['plant_id_pudl', 'utility_id_pudl', 'utility_id_ferc1']))
+        )
+
+    def _prep_steam_cleaned_ferc1(self):
+        self.steam_cleaned_ferc1 = (
+            pd.merge(
+                self.steam_cleaned_ferc1_raw,
+                self.connects_ferc1_eia[['record_id_ferc', 'plant_part']],
+                on=['record_id_ferc'],
+                how='left'
+            )
+            .pipe(pudl.helpers.convert_to_date)
+        )
+
+    def _prep_plant_parts_ordered(self):
+        pudl_settings = pudl.workspace.setup.get_defaults()
+        pudl_engine = sa.create_engine(pudl_settings["pudl_db"])
+        table_compiler = make_plant_parts_eia.CompileTables(
+            pudl_engine, freq='AS', rolling=True)
+        parts_compilers = make_plant_parts_eia.CompilePlantParts(
+            table_compiler, clobber=True)
+        self.plant_parts_ordered = parts_compilers.plant_parts_ordered
+
+    def prep_inputs(self):
+        """Prepare all inputs needed for connecting deprecation to FERC1."""
+        # the order here is important. We are preping the inputs needed
+        # for later inputs
+        self._prep_raw_tables()
+        self._prep_plant_parts_eia()
+        self._prep_connects_ferc1_eia()
+        self._prep_connects_deprish_eia()
+        self._prep_steam_cleaned_ferc1()
+
+        self._prep_plant_parts_ordered()
+
+
+def check_high_level_connections(
+        connects_all_deprish_ferc1,
+        steam_cleaned_ferc1):
```

Comment:
Some indication of what is "left" and what is "right" in the merge indicator would be helpful to understand things here.